action.yml

```diff
@@ -6,17 +6,28 @@ inputs:
     required: false
   channel:
     description: "The channel name to post to"
-    required: true    
+    required: true
+  thread_ts:
+    description: "The timestamp ID of the parent message to reply to"
+    required: false
   text:
     description: "The message text to post"
     required: true
+outputs:
+  ts:
+    description: "The timestamp ID of the message that was just posted"
 branding:
   icon: 'tag'
   color: 'blue'
 runs:
   using: 'composite'
   steps:
     - run: |
-        curl --silent --show-error -X POST -H "Authorization: Bearer ${{ env.SLACK_BOT_TOKEN || inputs.token }}" -H "Content-Type: application/json" --url https://slack.com/api/chat.postMessage \
-          -d "{\"channel\": \"${{ inputs.channel }}\", \"text\":\"${{ inputs.text }}\"}"
+        {
```

Comment:
Tested this using another token and posting to a private channel.  It's working great.